users/models.py

```diff
@@ -131,3 +131,17 @@ def save(self, **kwargs):
             self.last_vote_affirmation = timezone.now()
 
         return super().save(**kwargs)
+
+
+class UserGroup(models.Model):
+    name = models.CharField(max_length=255)
+    location = models.CharField(blank=True, null=True, max_length=255)
```

Comment:
I assume almost all user groups have physical locations so perhaps we should make this required?

Reply:
Ok, sure. This has been done.